components/script/dom/vrdisplay.rs

```diff
@@ -517,7 +518,12 @@ impl VRDisplay {
                 let task = Box::new(task!(handle_vrdisplay_raf: move || {
                     this.root().handle_raf(&sender);
                 }));
-                js_sender.send(CommonScriptMsg::Task(WebVREvent, task, Some(pipeline_id))).unwrap();
+                js_sender.send(CommonScriptMsg::Task(
+                    WebVREvent,
+                    task,
+                    Some(pipeline_id),
+                    TaskSourceName::DOMManipulation,
```

Comment:
Please add a NOTE asking whether long-term we want to be using the `DOMManipulation` task source here.